### PR TITLE
Contacts

### PR DESCRIPTION
**Pull request by kain88-de**
implement contact analysis from MDAnalysis

### CODE REVIEW

File: pmda/contacts.py [hunk 1]
```diff
@@ -0,0 +1,235 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+#
+# PMDA
+# Copyright (c) 2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+"""
+Contact analysis tools
```
Comment (orbeckst) [inline pmda/contacts.py]: Can you add minimal docs as in, e.g., pmda/rms.py?

File: pmda/contacts.py [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+from __future__ import absolute_import, division
+
+import MDAnalysis as mda
+from MDAnalysis.lib.distances import distance_array
+import numpy as np
+
+from .parallel import ParallelAnalysisBase
+
+
+def soft_cut_q(r, r0, beta=5.0, lambda_constant=1.8):
```
Comment (orbeckst) [inline pmda/contacts.py]: Do we have to duplicate all this code, can't we import it from `MDAnalysis.analysis.contacts`?

File: pmda/rms.py [hunk 3]
```diff
@@ -27,7 +27,7 @@ def __init__(self, mobile, ref):
         self._ref_pos = ref.positions.copy()
 
     def _prepare(self):
-        pass
+        self.rmsd = None
```
Comment (orbeckst) [inline pmda/rms.py]: You could have just submitted this patch directly to master...

**Comment by kain88-de** [comment]
@orbeckst can you review if everything is OK

**Pull request merged**